test/parallel/test-repl-top-level-await.js

```diff
@@ -160,7 +161,8 @@ async function ctrlCTest() {
     { ctrl: true, name: 'c' }
   ]), [
     'await timeout(100000)\r',
-    'Thrown: Error: Script execution interrupted.',
+    'Thrown: Error [ERR_SCRIPT_EXECUTION_INTERRUPTED]: ' +
+      errors.message('ERR_SCRIPT_EXECUTION_INTERRUPTED'),
```

Comment:
Since we are actually testing the error message, we don't really need to generate it with `errors.message`, can you change this to match the actual error message instead?